src/shared-components/icon/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import iconFilePath from '../../svgs/icons.svg';
+import { COLORS } from '../../constants';
+
+const propTypes = {
+  iconName: PropTypes.string.isRequired,
+  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  rotate: PropTypes.number,
+  fill: PropTypes.string,
+  inline: PropTypes.bool,
+  className: PropTypes.string,
+  duskId: PropTypes.string,
+};
+
+const defaultProps = {
+  className: '',
+  fill: COLORS.primary,
+  height: 15,
+  inline: false,
+  rotate: 0,
+  width: 15,
+};
+
+const Icon = ({
+  className,
+  fill,
+  height,
+  iconName,
+  inline,
+  rotate,
+  width,
+  duskId,
+  ...rest
+}) => {
+  // eslint-disable-next-line no-undef
+  const icon = window.__SVG_INLINE__
+    ? `#${iconName}`
+    : `${iconFilePath}#${iconName}`;
+  return (
+    <svg
+      width={width}
+      height={height}
+      fill={fill}
+      data-dusk-id={duskId}
```

Comment:
dont think we should have this in component library.

Reply:
I believe we can just add this during usage of the component and don't need to pass it in— `<Icon data-dusk-id="something" />`

Reply:
yup, I can definitely remove it. Was doing a lot of brainless copy pasta 🍝